Patterns/Japanese/Japanese-Numbers.yaml

```diff
@@ -264,14 +259,14 @@ LessRegex: !simpleRegex
   def: (小なり|より小さい|<)
 EqualRegex: !simpleRegex
   def: (等しい|イコール|=)
-MoreOrEqual: !nestedRegex
-  def: 大なりイコール|小さくない
+MoreOrEqual: !simpleRegex
+  def: (大なりイコール|小さくない)
 MoreOrEqualSuffix: !simpleRegex
-  def: より(大なりイコール|小さくない)
-LessOrEqual: !nestedRegex
-  def: 小なりイコール|小さくない
+  def: (より(大なりイコール|小さくない))
+LessOrEqual: !simpleRegex
```

Comment:
Don't change from nested to simple unless really required. MoreOrEqual is used in OneNumberRangeMoreRegex1, so it is nested.